packages/generator-reactserver/generators/app/templates/_package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "<%= name %>",
+  "version": "0.0.1",
+  "description": "An hello world example for react-server",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/redfin/example-react-server.git"
+  },
```

Comment:
> For GitHub, GitHub gist, Bitbucket, or GitLab repositories you can use the same shortcut syntax you use for npm install

https://docs.npmjs.com/files/package.json#repository

```json
  "repository": "redfin/example-react-server"
```

Though, admittedly, I don't actually know what that repository is...